src/main/java/org/jitsi/videobridge/AbstractEndpointMessageTransport.java

```diff
@@ -408,18 +412,21 @@ protected void onPinnedEndpointsChangedEvent(
      * {@code SelectedEndpointChangedEvent} which has been received.
      * @param newSelectedEndpoints the new selected endpoints
      */
-    protected void onSelectedEndpointsChangedEvent(
+    protected void selectedEndpointsChangedEvent(
         JSONObject jsonObject,
         Set<String> newSelectedEndpoints)
     {
         videoConstraintsCompatibility.setSelectedEndpoints(newSelectedEndpoints);
-        onVideoConstraintsChangedEvent(
+        senderVideoConstraintsChangedEvent(
             videoConstraintsCompatibility.computeVideoConstraints());
     }
 
-    protected void onVideoConstraintsChangedEvent(Map<String, VideoConstraints> newVideoConstraints)
+    protected void senderVideoConstraintsChangedEvent(Map<String, VideoConstraints> videoAllocationPolicyMap)
```

Comment:
Rename the variable to `videoConstraintsMap` and add a doc?

Reply:
Thanks!